app/root.jsx

```diff
@@ -7,6 +7,7 @@ import {
   ScrollRestoration,
 } from "remix";
 import AlbyHeadIcon from "../public/images/alby_icon_head_icon.png";
+import OGImage from "../public/images/og_image.svg";
```

Comment:
I think we should use an JPG or PNG here. I could imagine that SVGs are not everywhere supported.

Reply:
yep I think svg is not supported: https://indieweb.org/The-Open-Graph-protocol#Does_not_support_SVG_images
(interesting page btw.)